src/components/TodoList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { UserInfo } from './UserInfo';
+import { TodoInfo } from './TodoInfo';
+import { TodoWithUser } from '../types/TodoWithUser';
+
+export type Props = {
+  todosList: TodoWithUser[];
+};
+
+export const TodoList: React.FC<Props> = ({ todosList }) => (
+  <ul>
+    {todosList.map((todo: TodoWithUser) => (
+      <li key={todo.id}>
+        <TodoInfo
+          title={todo.title}
+          completed={todo.completed}
+        />
+        <UserInfo
+          name={todo.user?.name}
+          email={todo.user?.email}
+        />
```

Comment:
```suggestion
        {user && (
          <UserInfo
            name={todo.user.name}
            email={todo.user.email}
          />
        )
```
Можна використати conditional rendering і рендерити цей компонент тільки тоді, коли є юзер